src/nxt_cgroup.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) Andrew Clayton
+ * Copyright (C) F5, Inc.
+ */
+
+#ifndef _NXT_CGROUP_H_INCLUDED_
+#define _NXT_CGROUP_H_INCLUDED_
+
+#include <nxt_main.h>
+#include <nxt_process.h>
```

Comment:
Double blank after includes (and probably in other places I missed it).

See <https://nginx.org/en/docs/dev/development_guide.html#code_style>

Reply:
Yes and after the #define it seems... with the double spacing...

Reply:
Oh and between the typedefs and function prototypes...

Reply:
Yeah, and between function definitions.

Reply:
Hopefully I've got this right now.